libs/platform/src/lib/components/search-field/search-field.component.html

```diff
@@ -1,106 +1,62 @@
-                [ngClass]="{ 'fd-button--compact': compact }"
-                (click)="clearTextInput()"
-            ></button>
+                [ngClass]="{ 'fd-button--compact': compact }" (click)="clearTextInput()"></button>
         </span>
-        <span
-            class="fd-input-group__addon fd-input-group__addon--button"
-            [ngClass]="{ 'fd-input-group__addon--compact': compact }"
-            *ngIf="!isLoading"
-        >
+        <span class="fd-input-group__addon fd-input-group__addon--button"
+            [ngClass]="{ 'fd-input-group__addon--compact': compact }" *ngIf="!isLoading">
             <button
                 class="search-field--submit fd-input-group__button fd-button fd-button--transparent sap-icon--search"
-                [attr.id]="submitId"
-                [attr.disabled]="disabled ? '' : null"
-                [ngClass]="{ 'fd-button--compact': compact }"
-                (click)="onSearchSubmit()"
-            ></button>
+                [attr.id]="submitId" [attr.disabled]="disabled ? '' : null"
+                [ngClass]="{ 'fd-button--compact': compact }" (click)="onSearchSubmit()"></button>
         </span>
-        <span
-            class="fd-input-group__addon fd-input-group__addon--button"
-            [ngClass]="{ 'fd-input-group__addon--compact': compact }"
-            *ngIf="isLoading"
-        >
+        <span class="fd-input-group__addon fd-input-group__addon--button"
+            [ngClass]="{ 'fd-input-group__addon--compact': compact }" *ngIf="isLoading">
             <button
                 class="search-field--loading fd-input-group__button fd-button fd-button--transparent sap-icon--synchronize"
-                [ngClass]="{ 'fd-button--compact': compact }"
-                (click)="onSearchSubmit()"
-            ></button>
+                [ngClass]="{ 'fd-button--compact': compact }" (click)="onSearchSubmit()"></button>
         </span>
     </div>
 </div>
 <ng-template #suggestionMenuTemplate>
-    <div
-        class="search-field--dropdown fd-popover__body fd-popover__body--no-arrow"
-        [attr.dir]="dir"
-        *ngIf="(dropdownValues$ | async | suggestionMatches: inputText)?.length > 0"
-    >
-        <nav class="fd-menu" tabindex="-1" role="menu" [attr.id]="menuId" (keydown)="onKeydown($event)">
-            <ul class="fd-menu__list">
-                <li
-                    fdpSearchFieldSuggestion
-                    class="search-field--suggestion-item fd-menu__item"
-                    *ngFor="let value of dropdownValues$ | async | suggestionMatches: inputText"
-                    (keydown.enter)="onItemClick(value)"
-                    (click)="onItemClick(value)"
-                    [innerHTML]="value | highlight: inputText:true"
-                ></li>
+    <div class="search-field--dropdown fd-popover__body fd-popover__body--no-arrow" [attr.dir]='dir'
```

Comment:
I have mixed feelings about the `search-field--dropdown` naming. It's not prefixed by `fd` or `fdp` so the enduser will have no idea this class is coming from fundamentals. It's not following the BEM methodology. `--` is used for modifier classes like `--compact`. If it's an element it should be with `__`

Reply:
I will update the class names to `fdp-search-field__*`.

Reply:
@InnaAtanasova OK, I've updated the style names.